sql/README.md

```diff
@@ -18,6 +18,10 @@ Once you are ready to add a new query to the repository, open a pull request fol
 
 ## Query index
 
+### 2023/01
+
+* [impact of third-party render blocking scripts](./2023/01/third-party-render-blocking-script.sql)
```

Comment:
See above, maybe "Usage" instead of "Impact"? Nit-pick, but maybe also change the file name to `third-party-render-blocking-scripts-usage.sql`.

Reply:
Thanks for the suggestion. The SQL file name updated.